### PR TITLE
feat: add pull_request_target_branch support

### LINKED ISSUE #34: Add `base_branch` to exported envs
Currently it does not look like there's a way to get the base branch being targeted in the case of a pull request.

### PR DESCRIPTION
**Pull request by iamogbz**
Closes #34

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,6 +62,7 @@ if (process.env.TRAVIS) {
   event = 'push';
   commit_message = ''; // wercker does not expose commit message
   pull_request_number = ''; // wercker does not expose pull request number
+  pull_request_target_branch = ''; // wercker does not expose pr target branch
```
Comment (siddharthkp) [inline index.js:65]: should we leave them as not defined instead of empty string for strict typechecks?

Comment (iamogbz) [reply index.js:65]: I think empty strings are fine for two reasons, I believe env variables are empty strings by default, and it keeps with the existing pattern without introducing breaking changes.

Comment (siddharthkp) [reply index.js:65]: valid!

**Comment by siddharthkp** [comment]
Published in 1.14.0 🎉

**Comment by kuldeepkeshwar** [comment]
> Published in 1.14.0 🎉

@siddharthkp what if we automate this. After every release, the pull request gets labeled `v.1.14.9` 🙃

**Pull request merged**